bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/photon_proton_NLO_MG_pp_13TeV/TTbar_photon_proton_NLO_MG_pp_13TeV/photo_NLO_M_ttbar_pp_13TeV.txt

```diff
@@ -0,0 +1,10 @@
+#import model sm-no_b_mass
+generate p a > t t~ [QCD]
+set automatic_html_opening False
+output jam_lpp2_2_photo_NLO_ttbar_pp_13TeV
+launch
+launch
```

Comment:
please remove the launch lines. what is the idea behind them?

Reply:
This is the way it was provided by the developers, but in principle I would think we don't need them